### PR TITLE
change way of adding variant caller to vcf

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

- [x] change way of adding variant caller to vcf
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/genomic-medicine-sweden/tomte/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core pipelines lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Updated nf-core/tools template to v3.0.2 [#167](https://github.com/genomic-medicine-sweden/tomte/pull/167)
 - Updated multiqc version to 1.25.1 [#167](https://github.com/genomic-medicine-sweden/tomte/pull/167)
 - Updated modules bcftools/annotate, bcftools/merge, bcftools/mpileup, bcftools/norm, bcftools/stats, bcftools/view, cat/fastq, ensemblvep/filtervep, ensemblvep/vep, fastp, gatk4/asereadcounter, gatk4/bedtointervallist, gatk4/createsequencedictionary, gatk4/haplotypecaller, gatk4/splitncigarreads, gatk4/variantfiltration, gawk, gffcompare, gunzip, picard/collectinsertsizemetrics, picard/collectrnaseqmetrics, salmon/index, salmon/quant, samtools/faidx, samtools/index, samtools/view, star/align, star/genomegenerate, stringtie/stringtie, tabix/bgziptabix, tabix/tabix, ucsc/wigtobigwig, untar [#177](https://github.com/genomic-medicine-sweden/tomte/pull/177)
+- How variants are called is added to the vcf using the local module add_found_in_tag [#183](https://github.com/genomic-medicine-sweden/tomte/pull/183)
```
Comment (Lucpen) [inline CHANGELOG.md:42]:
```suggestion
- Changed how variants are caller is added to the vcf, it is now done using the local module add_found_in_tag [#183](https://github.com/genomic-medicine-sweden/tomte/pull/183)
```

**Comment by Lucpen** [comment]
@Jakob37 I have applied Felix's suggestion here :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### `Fixed`
 
 - Versions for all modules involving drop will now be outputed in version.yml and multiqc file [#174](https://github.com/genomic-medicine-sweden/tomte/pull/174)
-  Fixed bug when running variant calling with gatk [[#182](https://github.com/genomic-medicine-sweden/tomte/pull/182)]
+- Fixed bug when running variant calling with gatk [[#182](https://github.com/genomic-medicine-sweden/tomte/pull/182)]
```
Comment (fellen31) [inline CHANGELOG.md:19]:
```suggestion
- Fixed bug when running variant calling with gatk [#182](https://github.com/genomic-medicine-sweden/tomte/pull/182)
```

**Review by fellen31 — COMMENTED** [review]
Is bcftools annotate used anywhere expect here? Else it could be removed completely.

**Comment by Lucpen** [comment]
Good point @fellen31 , I have now removed it

**Review by fellen31 — APPROVED** [review]
LGTM! We could think about putting shared local modules (and subworkflows) in a gms repository to make them easier to maintain.

**Pull request merged**